src/idl_gen_pybind.cpp

```diff
@@ -876,8 +922,8 @@ class PybindGenerator : public BaseGenerator {
         code_ += Indent(3) +
                  "[](const {{CPP_TYPE}} &self) -> "
                  "std::optional<{{CPP_FIELD_TYPE}}*> {";
-        code_ +=
-            Indent(4) + "if (self.{{CPP_FIELD}} == nullptr) { return std::nullopt; }";
```

Comment:
are formatting changes intended?

Reply:
Yeah, clang-format seems to have changed its mind on this. Haven't investigated why, but I'll see if these spurious formatting changes happen again.